production_configs/20240828_v0.10.9_GC_dec_min_2924_tuned/lstmcpipe_config_2024-08-28_PathConfigAllSkyFull.yaml

```diff
@@ -0,0 +1,3048 @@
+# lstmcpipe generated config from PathConfigAllSkyFull - 2024-08-28
+
+workflow_kind: lstchain
+
+# prod_id ex: local_no_n_islands. Default; v00 (if key left empty or None)
+prod_id: 20240828_v0.10.9_GC_dec_min_2924_tuned
+
+source_environment:
+  source_file: /fefs/aswg/software/conda/etc/profile.d/conda.sh
+  conda_env: lstchain-v0.10.7
```

Comment:
this needs to be changed to lstchain-v0.10.9

Reply:
I have updated the config file by changing the lstchain environment